src/physics/material/MaterialParams.i.hh

```diff
@@ -20,7 +20,7 @@ const std::string& MaterialParams::id_to_label(ElementDefId el) const
 
 //---------------------------------------------------------------------------//
 /*!
- * Get the label for an material.
+ * Get the label for a material. 
```

Comment:
A stray trailing space got added here -- maybe need to rerun clang-format on this the commits?